docs/oathkeeper/sdk/api.md

```diff
@@ -657,9 +657,9 @@ Status Code **200**
 | »» match                                                                    | [ruleMatch](#schemarulematch)       | false    | none         | none                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
For future reference - this will also be auto-generated from swagger spec :) See warning at the top:

> 

    You are viewing a REST API documentation. This documentation is auto-generated from a swagger specification which itself is generated from annotations in the source files of the project. It is possible that this documentation includes bugs and that code samples are incomplete or wrong.

> If you find issues in the respective documentation, please do not edit the markdown files directly (as they are generated) but raise an issue on the project's GitHub instead. This documentation will improve over time with your help! If you have ideas how to improve this part of the documentation, feel free to share them in a GitHub issue any time.


You can leave this as is however - it will be automatically updated once we merge oathkeeper pr

Reply:
yeah, my bad. I was mindlessly hunting those mutators 😅

Reply:
all good :)